components/file_system/src/lib.rs

```diff
@@ -25,26 +29,28 @@ use std::sync::{Arc, Mutex};
 use openssl::error::ErrorStack;
 use openssl::hash::{self, Hasher, MessageDigest};
 
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug)]
 pub enum IOOp {
     Read,
     Write,
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq)]
 pub enum IOType {
-    Other,
-    Read,
-    Write,
-    Coprocessor,
-    Flush,
-    Compaction,
-    Replication,
-    LoadBalance,
-    Import,
-    Export,
+    Other = 0,
+    Read = 1,
+    Write = 2,
```

Comment:
I want to change these two types cause it's confusing with IOOp. Maybe call it `Storage` and `Raftstore`?

Reply:
As per the original proposal, `IOType::Read` refers to foreground read query (I believe we have read request going through raftstore too?), and `IOType::Write` refers to write transaction (kvdb WAL included). So `Storage` and `Raftstore` isn't ideal either.

Reply:
I think the confusion part exists but is manageable, because `IOOp` is only exposed to core components related to file_system.